f2/src/utils/formatAnalystEmail.js

```diff
@@ -30,6 +30,7 @@ const formatReportInfo = data => {
 
   delete data.reportId // we delete the parts of the data object that we've displayed
   delete data.submissionTime // so that at the end we can display the rest and ensure that
+  delete data.submissionDate //submissionDate does not needed in analyst report
   delete data.language // we didn't miss anything
```

Comment:
The three lines of comment before were meant to be read as one long comment, so adding the new one in the middle makes things a bit confusing 😄  More confusing, perhaps, is why we did the three line comment like this in the first place. Could you move those three lines to before all these delete lines? Leaving in the line you added here just as it is.